met-web/src/routes/NoResults.tsx

```diff
@@ -0,0 +1,120 @@
+import { Grid, Toolbar, Box } from '@mui/material';
+import { MetParagraph, HeaderTitle, SubHeader, ListItem } from 'components/common';
+import React from 'react';
+import { useAppTranslation } from 'hooks';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCirclePhone } from '@fortawesome/pro-solid-svg-icons/faCirclePhone';
+import { faCircleEnvelope } from '@fortawesome/pro-solid-svg-icons/faCircleEnvelope';
+
+const marginStyle = { mr: 2 };
+
+const SuggestionsList = ({ translate }: { translate: (key: string) => string }) => (
+    <Box>
+        <SubHeader>{translate('NoResults.paragraph')}</SubHeader>
+        <ul>
+            <li>
+                <ListItem>{translate('NoResults.list.0')}</ListItem>
+            </li>
+            <li>
+                <ListItem>{translate('NoResults.list.1')}</ListItem>
+            </li>
+            <li>
+                <ListItem>{translate('NoResults.list.2')}</ListItem>
+            </li>
+        </ul>
+    </Box>
+);
+
+const NoResults = () => {
+    const { t: translate } = useAppTranslation();
+
+    return (
+        <>
+            <Toolbar />
+            <Grid
+                container
+                direction={'column'}
+                justifyContent="left"
+                alignItems="left"
+                spacing={1}
+                padding={'2em 2em 1em 1em'}
+            >
+                <Grid item sx={{ ...marginStyle, marginBottom: 3 }}>
+                    <HeaderTitle data-testid="NoResultsHeader">{translate('NoResults.header')}</HeaderTitle>
+                </Grid>
+                <Grid item xs={6} justifyContent={'left'} sx={{ ...marginStyle, marginBottom: 3 }}>
+                    <SuggestionsList translate={translate} />
+                </Grid>
+                <Grid
+                    container
+                    direction="column"
+                    alignItems="center"
+                    padding={0}
+                    width="100%"
+                    maxWidth="1120px"
+                    height="auto"
+                    border="1px solid #FAF9F8"
+                    boxShadow="0px 0.6px 1.8px rgba(0, 0, 0, 0.1), 0px 3.2px 7.2px rgba(0, 0, 0, 0.13)"
+                    borderRadius="8px"
+                    display={'none'} // TODO: remove this once contact information is decided
```

Comment:
I appreciate that you want to save us some work with the contact area, but I'm hesitant to add this code because of the uncertainty. If we decide not to have a contact area, we'll have to go back and remove all this.

Did you speak to Jess or the PO about this? Did they instruct you to add it anyway? If we don't have explicit direction from the PO on this, then I'd recommend we remove it.

Reply:
I did talk to Jess about it and she suggested to add the contact and make it hidden as is mentioned in the ticket as well.

Reply:
Thanks for confirming you spoke with someone about this. This feels like one of those situations where we, as developers, need to decide what's best for the code. Technically, the PO can let us know what's needed for the user, but how we do that is up to us. From a maintainability standpoint, my instinct is usually not to build things on the chance that we use them again, but to build as needed.

That said, it's not that much code to leave in, and hopefully the PO and the designer have a plan for it in the short term. We can proceed with it.

Reply:
Thanks will follow this going forward.